bin/process-csv.js

```diff
@@ -0,0 +1,163 @@
+                        categories: categoryMap[record.Category.trim()] || 'Default Category/Books/Other',
+                        product_websites: 'base',
+                        name: title,
+                        description: record.Description,
+                        short_description: '',
+                        weight: 1,
+                        product_online: 1,
+                        tax_class_name: 'Taxable Goods',
+                        visibility: 'Catalog, Search',
+                        price: record.RRP,
+                        special_price: '',
+                        special_price_from_date: '',
+                        special_price_to_date: '',
+                        url_key: slug(title) + '-' + record.BookCode,
+                        meta_title: title,
+                        meta_keywords: [ 'Western Australia', 'History', record.Category.toLowerCase() ].join(', '), // TODO Others? Author?
+                        meta_description: record.Description,
+                        base_image: imageFilename,
+                        base_image_label: imageFilename ? 'Cover of ' + title : '',
+                        small_image: imageFilename,
+                        small_image_label: imageFilename ? 'Cover of ' + title : '',
+                        thumbnail_image: imageFilename,
+                        thumbnail_image_label: imageFilename ? 'Cover of ' + title : '',
+                        created_at: (new Date(Math.min(
+                            record.FirstOrderDate ? Date.parse(record.FirstOrderDate) : Date.now(),
+                            record.LastSaleDate ? Date.parse(record.LastSaleDate) : Date.now(),
+                            record.LastUpdate ? Date.parse(record.LastUpdate) : Date.now(),
+                            record.StocktakeDate ? Date.parse(record.StocktakeDate) : Date.now()
+                        ))).toISOString(),
+                        updated_at: record.LastUpdate ? (new Date(record.LastUpdate)).toISOString() : Date.now(),
+                        new_from_date: '',
+                        new_to_date: '',
+                        display_product_options_in: 'Block after Info Column',
+                        map_price: '',
+                        msrp_price: record.RRP,
+                        map_enabled: '',
+                        gift_message_available: '',
+                        custom_design: '',
+                        custom_design_from: '',
+                        custom_design_to: '',
+                        custom_layout_update: '',
+                        page_layout: '',
+                        product_options_container: '',
+                        msrp_display_actual_price_type: 'Use config',
+                        // TODO Can we assume this? Or should it be blank?
+                        country_of_manufacture: 'Australia',
+                        // TODO Derive correct value for `quantity_and_stock_status`
```

Comment:
Oops I commented on the wrong line. I was talking about Country. But I don't think we can assume even that (many books have 'publisher X, London ' but printed in China,  so country of manufacture could easily not be Australia. But yes TODO caters for it.